models/application.js

```diff
@@ -36,7 +37,15 @@ module.exports = function(sequelize, DataTypes) {
             }
         },
         {
-            freezeTableName: true
+            freezeTableName: true,
+            getterMethods: {
+                grantAmount() {
+                    if (this.application_data && this.application_data['project-budget']) {
```

Comment:
This makes a lot of sense _but_ I'm conscious that this is specific to the simple funding form, might not apply to _all_ applications. Just noting, not sure we know the answer to this just yet.

Reply:
Yeah... will leave it there for now but definitely will come up in future.